File: ABC/263/a.cpp
#include <bits/stdc++.h>
using namespace std;

int a[14];

int main(){
    for(int i=0;i<5;i++){
        int tmp;
        cin >> tmp;
        a[tmp]++;
    }
    int two = 0, three = 0;
    for(int i=0;i<14;i++){
        if(a[i]==2)two++;
        if(a[i]==3)three++;
    }
    if(two>0&&three>0){
        cout << "Yes" << endl;
    }else{
        cout << "No" << endl;
    }
}

File: ABC/263/a.java
import java.util.*;

public class a {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int[] arr = new int[14];
        for(int i=0;i<5;i++){
            arr[sc.nextInt()]++;
        }
        sc.close();
        int three = 0, two = 0;
        for(int i=0;i<arr.length;i++){
            if(arr[i]==3){
                three++;
            }
            else if(arr[i]==2){
                two++;
            }
        }
        if(three>0&&two>0){
            System.out.println("Yes");
        }
        else{
            System.out.println("No");
        }
    }
}